lib/player.ex

```diff
@@ -0,0 +1,27 @@
+defmodule Player do
+  defstruct [:marker]
+
+  def get_move(marker, board) do
+    get_valid_move(marker, board)
+  end
+
+  def get_valid_move(marker, board) do
+    get_cell_location(marker)
+    |> Validator.is_valid_move(board)
+    |> get_valid_location(marker, board)
+  end
+
+  def get_valid_location(is_valid, _marker, _board) when elem(is_valid, 0) == true do
+    elem(is_valid, 1)
+  end
+
+  def get_valid_location(is_valid, marker, board) when elem(is_valid, 0) == false do
+    :cell_filled |> Messages.get() |> Console.display()
```

Comment:
I am thinking maybe this logic to respond to the user can stay in the Validator module, so that you wouldn't have to have this function in the Player module.

Reply:
Hmm - I updated the player class and removed this function. The player now only has `get_move`